src/CoreBundle/Service/MaintenanceModeService.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace ChameleonSystem\CoreBundle\Service;
```

Comment:
I'm not happy about the Service namespace and would like to use it as little as possible. Could we use a sub-namespace of CoreBundle/Maintenance?

Reply:
ok; Service->MaintenanceMode (Maintenance already exists with a different meaning (?))